src/content/fr/fundamentals/design-and-ui/media/images.md

```diff
@@ -403,7 +403,7 @@ Pour prendre la bonne décision, commencez par appliquer ces quelques directives
 
 Il est possible de réduire sensiblement la taille du fichier image en le soumettant à un post-traitement après l'avoir enregistré. Il existe tout un éventail d'outils destinés à la compression d'images : avec et sans perte, en ligne, avec interface graphique, par ligne de commande, etc. Lorsque cela s'avère possible, il est conseillé d'automatiser l'optimisation des images, de sorte qu'elle soit considérée comme un objet de première classe dans votre flux de travail.
 
-Plusieurs outils permettent d'effectuer une compression sans perte plus poussée sur les fichiers `JPG` et `PNG`, sans nuire à la qualité d'image. Pour le format `JPG`, essayez [jpegtran](http://jpegclub.org/) ou [jpegoptim](http://freshmeat.net/projects/jpegoptim/) (disponible seulement sur Linux ; à exécuter avec l'option --strip-all). Pour le format `PNG`, essayez [OptiPNG](http://optipng.sourceforge.net/) ou [PNGOUT](http://www.advsys.net/ken/util/pngout.htm).
+Plusieurs outils permettent d'effectuer une compression sans perte plus poussée sur les fichiers `JPG` et `PNG`, sans nuire à la qualité d'image. Pour le format `JPG`, essayez [jpegtran](http://jpegclub.org/){: .external } ou [jpegoptim](http://freshmeat.net/projects/jpegoptim/) (disponible seulement sur Linux ; à exécuter avec l'option --strip-all). Pour le format `PNG`, essayez [OptiPNG](http://optipng.sourceforge.net/) ou [PNGOUT](http://www.advsys.net/ken/util/pngout.htm).
```

Comment:
@jpmedley - can you send me the script you're using to do this, it seems to miss several external links (see above) and sometimes adds extra links.